app/views/helpers/calendars/jalali/jdf.php

```diff
@@ -0,0 +1,647 @@
+<?php /* In the name of Allah = بسم اللّه الرّحمن الرّحیم */
+
+/**
+ * @فارسی : توابع زمان و تاریخ هجری شمسی (جلالی) در پی اچ پی
+ * @name: Hijri_Shamsi,Solar(Jalali) Date and Time Functions
+ * @Author : Reza Gholampanahi & WebSite : http://jdf.scr.ir
+ * @License: GNU/LGPL _ Open Source & Free : [all functions]
+ * @Version: 2.76 =>[ 1399/11/28 = 1442/07/04 = 2021/02/16 ]
+ */
+
+/*	F	*/
+function jdate($format, $timestamp = '', $none = '', $time_zone = 'Asia/Tehran', $tr_num = 'fa') {
```

Comment:
Is there a version of this library (or equivalent) that exists on https://packagist.org so that we could depend on it though `composer`?

Reply:
E.g. one of https://packagist.org/?query=Jalali ?